### ISSUE TITLE
`one_or_more(byte_range('b-f'))` only accepts two characters

### ISSUE THREAD

**Issue by hudson-ai** [hudson-ai/guidance#4]
With a ByteTokenizer, the following grammar can only accept 1-2 bytes from the range `b-f`, e.g. "b", "bb", "eg", ...
```python
one_or_more(byte_range('b-f'))
```

Equivalent ll_grammar: 
```
{'grammars': [{'greedy_lexer': False,
   'nodes': [{'Select': {'among': [1, 2]}},
    {'Join': {'sequence': [0, 2]}},
    {'Gen': {'body_rx': 0, 'stop_rx': '', 'temperature': None}}],
   'rx_nodes': [{'ByteSet': [0, 0, 0, 124, 0, 0, 0, 0]}]}]}
```

More specifically, after processing two bytes/tokens, calling `mid_process` on the `LLInterpreter` gives `stop=True` in the response.

This seems to only happen with gen nodes in the one_or_more, as normal literals, etc. are unproblematic.

**Comment by mmoskal** [comment]
I check in a fix, but I'm not sure if it doesn't break something else...

**Comment by hudson-ai** [comment]
This seems fixed, but the behavior of #7 just changed a bit. For example, rather than requiring a single whitespace character at the end of json generation, we now require two characters... 😆
